dae-alright/src/components/modal/custom-modal/custom-modal.component.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import Modal from "react-bootstrap/Modal";
+import { Button } from "react-bootstrap";
+import "./custom-modal.styles.scss";
+
+const CustomModal = props => {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <>
+      <Button variant="primary" onClick={handleShow}>
+        Sign-Up now
+      </Button>
```

Comment:
Remove this button because it will be in whichever components that will be using this

Reply:
I have removed this